charts/ceph-csi-cephfs/values.yaml

```diff
@@ -275,6 +275,26 @@ secret:
   adminID: <plaintext ID>
   adminKey: <Ceph auth key corresponding to ID above>
 
+# This is a sample configmap that helps define a Ceph configuration as required
+# by the CSI plugins.
+# Sample ceph.conf available at
+# https://github.com/ceph/ceph/blob/master/src/sample.ceph.conf Detailed
+# documentation is available at
+# https://docs.ceph.com/en/latest/rados/configuration/ceph-conf/
+cephconf: |
+  [global]
+    auth_cluster_required = cephx
+    auth_service_required = cephx
+    auth_client_required = cephx
+
+    # Workaround for http://tracker.ceph.com/issues/23446
+    fuse_set_user_groups = false
+
+    # ceph-fuse which uses libfuse2 by default has write buffer size of 2KiB
+    # adding 'fuse_big_writes = true' option by default to override this limit
+    # see https://github.com/ceph/ceph-csi/issues/1928
+    fuse_big_writes = true
```

Comment:
we are explicitly setting this mount option while fuse mounter is used via https://github.com/ceph/ceph-csi/pull/1959, so this is not valid either?